gitbook-docs/data_model.md

```diff
@@ -264,6 +264,42 @@ the value should be merged to the full model mounted where the delta‘s context
 }
 ```
 [<]: #
+## Delta Format For Metadata
+
+Metadata can also be updated via a delta within the `meta` key.
+
+Since meta data is not often updated it is only sent when there has been a change. See [Subscription Protocol](subscription_protocol.md) for details.
+
+[>]: # (mdpInsert ```json fsnip ../samples/delta/docs-data_model_meta_deltas.json --prettify 2 20)
+```json
+{
+  "context": "vessels.urn:mrn:imo:mmsi:234567890",
```

Comment:
Metadata is about a path irrespective of the particular wildcard value, so the vessel id does not really work here or is plain misleading. At least that is how the current metadata works in node server and in the js "sdk".

I suggest we denote the id part of the path with `*`.

Ditto for stuff like `propulsion.*`.

Another solution is to forego `context` all together and just deal with paths with wildcards. This approach has worked underneath the hood when filling in metadata into the HTTP api responses from the `keyswithmetadata.json` file.

This is probably the main question in this PR, so I'll stop my review here for the time being.